__tests__/topaz.ts

```diff
@@ -20,7 +20,7 @@ export class Topaz {
     await execute(
       "topaz configure -r ghcr.io/aserto-policies/policy-todo:2.1.0 -n todo -d -f"
     );
-    await execute("topaz start");
+    await execute("topaz start --container-version=model-v2.3");
```

Comment:
Could you please prepare a separate PR the reverts this change. We'll merge it once topaz 0.31 is released.

Reply:
https://github.com/aserto-dev/aserto-node/pull/68